department-of-veterans-affairs/abd-vro
# Feature/mcp 1874 user fixes

# MCP 1874 Bug Fixes

## Description



### What was the problem?

Postgres super user and flyway user were the same.

### How does this fix it?

Separates postgres and flyway users.

### Jira Tickets



- [MCP-1874](https://amida.atlassian.net/browse/MCP-1874)

## How to test this PR

- Also get new secrets from abd-vro-dev-secrets.
- Build and make sure application works (health assessment + pdf generation)

## Reminders

- Review [Pull-Requests](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests) guidelines
[Roadmap](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Roadmap) wiki pages
